acceptance-tests.md

```diff
@@ -66,10 +79,12 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Set and modify quotas for projects**
     * _As an administrator of the cluster, I should be able to set and modify compute, storage and object counts quotas for any project._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. For modifying attributes, allocation change requests can be requested by navigating to the active allocation.
         2. From here, a PI can approve the request and a call to the acct-mgt service will be made.
             1. For setting attributes, adding a new allocation attribute triggers a call to the acct-mgt service endpoint “/projects/{project_id}/quota".
+    * Acceptance tests:
+        1. You can explore quotas from within the Observability dashboard. For a given project named `myproject`, and resource `limits.cpu` for example, [Click here to check the "value" for the type=hard (max limit) and type=used (current value)](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22kube_resourcequota%7Bcluster%3D%5C%22nerc-ocp-prod%5C%22,namespace%3D%5C%22myproject%5C%22,resource%3D%5C%22limits.cpu%5C%22%7D%22%7D%5D).
 
```

Comment:
The acceptance test should walk through the change on the ColdFront side, verify that, and then the instructions about how to see the appropriate quota has changed in Obeservatorium are the OpenShift side.   Same approach as the previous use cases that are spread across two tools.

Reply:
revisions made